scripts/reformat_cpp.sh

```diff
@@ -18,7 +18,8 @@ TOPDIR="$SCRIPT_DIR/.."
 for i in "$TOPDIR"/include/proj/*.hpp "$TOPDIR"/include/proj/internal/*.hpp \
          "$TOPDIR"/src/iso19111/*.cpp "$TOPDIR"/test/unit/*.cpp "$TOPDIR"/src/apps/projinfo.cpp \
          "$TOPDIR"/src/tracing.cpp "$TOPDIR"/src/grids.hpp "$TOPDIR"/src/grids.cpp \
-         "$TOPDIR"/src/filemanager.hpp "$TOPDIR"/src/filemanager.cpp ; do
+         "$TOPDIR"/src/filemanager.hpp "$TOPDIR"/src/filemanager.cpp \
+         "$TOPDIR"/src/sqlite3.hpp "$TOPDIR"/src/sqlite3.cpp ; do
```

Comment:
Shouldn't this be `"$TOPDIR"/src/sqlite3_utils.hpp "$TOPDIR"/src/sqlite3_utils.cpp`?

Reply:
Good catch. Updated